test/unit/exports-wildcard/input.js

```diff
@@ -0,0 +1,2 @@
+import { y } from 'y/no';
```

Comment:
Does this code even run? It looks like `no.js` exports `x`, not `y` so how can you import `x` here?

Reply:
The exports wildcard remapping maps it to "node.js".